### PR TITLE
Support for fNIRS channels.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/3539

### CODE REVIEW

**Comment by wmvanvliet** [comment]
looks pretty comprehensive. I'm excited for seeing fNIRS support in MNE.

File: mne/channels/channels.py [hunk 1]
```diff
@@ -208,7 +211,9 @@ def compensation_grade(self):
                'stim': FIFF.FIFF_UNIT_NONE,
                'syst': FIFF.FIFF_UNIT_NONE,
                'bio': FIFF.FIFF_UNIT_V,
-               'ecog': FIFF.FIFF_UNIT_V}
+               'ecog': FIFF.FIFF_UNIT_V,
+               'hbo': FIFF.FIFF_UNIT_NONE,
+               'hbr': FIFF.FIFF_UNIT_NONE}
```
Comment (Eric89GXL) [inline mne/channels/channels.py]: I thought we created units for these as part of the defs, no?

File: mne/io/tests/test_pick.py [hunk 2]
```diff
@@ -152,8 +164,7 @@ def _check_fwd_n_chan_consistent(fwd, n_expected):
 
 @testing.requires_testing_data
 def test_pick_forward_seeg_ecog():
-    """Test picking forward with SEEG and ECoG
-    """
+    """Test picking forward with SEEG and ECoG.    """
```
Comment (Eric89GXL) [inline mne/io/tests/test_pick.py]: no need for spaces at the end

**Comment by agramfort** [comment]
looks really clean to me

@mcaffini can you check if it does the job for you?

**Comment by mcaffini** [comment]
Thanks everyone! @agramfort I will check and report here as soon as I can. :-)

File: mne/channels/channels.py (hunk 1 above)
Comment (jaeilepp) [inline mne/channels/channels.py]: It wasn't on the list, but we could use ``FIFF.FIFF_UNIT_MOL = 6  # mole``

Comment (Eric89GXL) [inline mne/channels/channels.py]: Sounds good to me

**Comment by Eric89GXL** [comment]
@mcaffini will you have time by the end of the week? If not, we can always test it later, but we don't want to hold up putting this in because we have a release coming up very soon.

**Comment by mcaffini** [comment]
👍 @Eric89GXL, I'll test tomorrow morning.

**Comment by mcaffini** [comment]
Hi everyone, test successful 😄

**Comment by Eric89GXL** [comment]
Thanks for checking @mcaffini

**Pull request merged**